src/test/java/domain/LadderGameTest.java

```diff
@@ -0,0 +1,60 @@
+package domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayNameGeneration;
+import org.junit.jupiter.api.DisplayNameGenerator;
+import org.junit.jupiter.api.Test;
+
+@DisplayNameGeneration(DisplayNameGenerator.ReplaceUnderscores.class)
+public class LadderGameTest {
+
+    /**
+     * ako   split     ash |-------|       | |-------|       | |-------|       | |-------|       | |-------|       | 꽝    5000       꽝
+     */
+    @Test
+    void 사다리_게임을_실행하면_사다리_결과를_반환한다() {
+        //given
+        List<Player> player = List.of(new Player("ako"), new Player("split"), new Player("ash"));
+        Players players = new Players(player);
+        int height = 5;
+        TestGenerator testGenerator = setUpTestGenerator();
+        Ladder ladder = Ladder.generateLadder(height, players, testGenerator);
+        List<Prize> prize = List.of(new Prize("꽝"), new Prize("5000"), new Prize("꽝"));
+        Prizes prizes = new Prizes(players.getPlayersSize(), prize);
+        LadderGame ladderGame = new LadderGame(players, ladder, prizes);
```

Comment:
```suggestion
        Players players = createPlayers();
        Prizes prizes = createPrizes();
        Ladder ladder = createLadder(setUpTestGenerator(), players);
        
        LadderGame ladderGame = new LadderGame(players, ladder, prizes);
```

다음처럼 셋업을 정리할 수 있을 것 같습니다. 테스트와 상관 없는 정보가 섞여있어서 어떤 행위가 테스트 되야하는지 잘 보이지 않네요

Reply:
정리해서 가독성 높게 관리하겠습니다!